### ISSUE TITLE
Add and/or fix noscript fallback for `organisms-search-menu`

### ISSUE THREAD

**Issue by laurenhamel** [emory-libraries/Pattern-Library#280]
This issue is in response to a [previous comment](https://github.com/emory-libraries/Pattern-Library/issues/204#issuecomment-513836823) on #204. 

## Testing

`pattern-library` [organisms-search-menu](http://template.library.emory.edu/styleguide/patternlibrary/current/?p=organisms-search-menu)

**Comment by laurenhamel** [comment]
@nikdragovic, this is ready for acceptance testing, although you won't really be able to see the changes that were made without disabling JavaScript. If you wish to do so, the easiest way will be to download a browser extension that allows you to toggle JavaScript on/off with one-click. Do note, though, that if you're testing without JS, you'll need to open the pattern in a separate window first: Click the settings icon in the top right corner of the navigation bar, then select "Open Pattern in New Window." Also worth noting, I did notice that some of the tab functionality for the JS-enabled version of the pattern was lost after making these changes, specifically the active state. I've added a fix into #201 to correct this, which is currently awaiting testing.
